### PR TITLE
change isort configuration

### PR DESCRIPTION
**Pull request by tardyp**
move future library on top of the imports
put twisted+zope+autobahn appart from the other 3rd party

We discussed on the stringio PR that we would put the future at the same place as the stdlib.
I wasn't able to do that, isort will always put the __future__ import at the same place as the future imports.
I think actually it is better to put future at the same place as __future__ (__future__ is required to be always first by python anyway)

### CODE REVIEW

File: .isort.cfg [hunk 1]
```diff
@@ -1,6 +1,9 @@
 [settings]
 line_length=110
-known_third_party=twisted,migrate,sqlalchemy,ldap3,klein,future,requests,MySQLdb,zope,coverage,autobahn,pkg_resources,jinja2,mock,dateutil,sphinx,setuptools
+known_future_library=future
+known_twisted=twisted,zope,autobahn,klein
+known_third_party=migrate,sqlalchemy,ldap3,txrequests,requests,MySQLdb,coverage,jinja2,mock,dateutil,sphinx,setuptools
 known_first_party=buildbot,buildbot_worker
 force_single_line=1
+sections=FUTURE,FUTURE_LIBRARY,STDLIB,TWISTED,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
```
Comment (sa2ajj) [inline .isort.cfg]: I'd change the order of `TWISTED` and `THIRDPARTY`

Comment (tardyp) [inline .isort.cfg]: ok

File: .isort.cfg [hunk 2]
```diff
@@ -1,6 +1,9 @@
 [settings]
 line_length=110
-known_third_party=twisted,migrate,sqlalchemy,ldap3,klein,future,requests,MySQLdb,zope,coverage,autobahn,pkg_resources,jinja2,mock,dateutil,sphinx,setuptools
+known_future_library=future
+known_twisted=twisted,zope,autobahn,klein
+known_third_party=migrate,sqlalchemy,ldap3,txrequests,requests,MySQLdb,coverage,jinja2,mock,dateutil,sphinx,setuptools
```
Comment (sa2ajj) [inline .isort.cfg]: I'd put `mock` into a separate group and place it *after* `_thirdparty`.

After that I am *happy*! :smile:

File: .isort.cfg [hunk 3]
```diff
@@ -1,6 +1,9 @@
 [settings]
 line_length=110
-known_third_party=twisted,migrate,sqlalchemy,ldap3,klein,future,requests,MySQLdb,zope,coverage,autobahn,pkg_resources,jinja2,mock,dateutil,sphinx,setuptools
+known_future_library=future
+known_twisted=twisted,zope,autobahn,klein
```
Comment (sa2ajj) [inline .isort.cfg]: [nit] zope, autobah, klein, twisted

Comment (sa2ajj) [inline .isort.cfg]: aha, thanks for the reminder.

then order of 3rd party and twisted + mock after 3rd party are the only things standing between me and +2.

Comment (tardyp) [inline .isort.cfg]: done

**Comment by sa2ajj** [comment]
@tardyp thanks again

**Pull request merged**